tests/framework/jailer.py

```diff
@@ -49,6 +50,7 @@ def __init__(
             daemonize=True,
             new_pid_ns=False,
             cgroups=None,
+            cgroup_ver=None,
```

Comment:
Would it make sense to set `cgroup_ver` to v1, as this is the default implementation?

Reply:
Left the default to None and added a test that does not set the parameter and expects the jailer to use cgroup-v1.